packages/sdk/src/sdk/services/Solana/programs/ClaimableTokensClient/ClaimableTokensClient.ts

```diff
@@ -197,7 +200,7 @@ export class ClaimableTokensClient {
   async createTransferInstruction(params: CreateTransferRequest) {
     const {
       feePayer: feePayerOverride,
-      ethWallet,
+      ethWallet = (await this.audiusWalletClient.getAddresses())[0]!,
```

Comment:
Sooooo, should we not just read it and check it properly and throw an exception if it doesn't exist?

Reply:
Yes! Great catch, lazy programming